runtime/onert/core/src/ir/LoweredGraph.cc

```diff
@@ -123,6 +124,9 @@ LoweredGraph::LoweredGraph(const Graph &graph, const compiler::CompilerOptions &
     pass::PermutationInsertionPass pi_pass(*this);
     pi_pass.run();
 
+    pass::PermutationEliminationPass pe_pass(*this);
+    pe_pass.run();
```

Comment:
All permute operations are inserted from `PermutationInsertionPass`, now it is time to remove some.